src/test/groovy/LibraryNameTest.groovy

```diff
@@ -232,4 +232,18 @@ class LibraryNameTest extends Specification {
         where:
         baseVersion << ['1', 'MOCOM-353', 'Test.Version', '1.Something.0', '1.2.3.4']
     }
+
+    @Unroll
+    def "Library version is correct for arbitrary commit in HEAD branch: #HEAD"() {
```

Comment:
You don't need @Unroll and #HEAD here. That's only for parameterized tests (i.e. tests that have a where: section at the end).